clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2378,12 +2404,18 @@ cir::CallOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
 
 ::mlir::ParseResult CallOp::parse(::mlir::OpAsmParser &parser,
                                   ::mlir::OperationState &result) {
-  return parseCallCommon(parser, result);
+
+  return parseCallCommon(parser, result, [](::mlir::OperationName name) -> ::mlir::StringAttr {
+      return getExtraAttrsAttrName(name);
```

Comment:
Passing a lambda here seems to be making this more complicated to read than necessary, compute `getExtraAttrsAttrName(result.name)` before calling and pass a `StringRef` instead. Alternatively, you can change `parseCallCommon` to directly use `"extra_attrs"`, not as clean, but acceptable given they both share the same tablegen class.